src/libs/Navigation/Navigation.ts

```diff
@@ -56,7 +58,7 @@ function getActiveRouteIndex(stateOrRoute: StateOrRoute, index?: number): number
         return getActiveRouteIndex(childActiveRoute, stateOrRoute.state.index ?? 0);
     }
 
-    if ('name' in stateOrRoute && stateOrRoute.name === NAVIGATORS.RIGHT_MODAL_NAVIGATOR) {
+    if ('name' in stateOrRoute && (stateOrRoute.name === NAVIGATORS.RIGHT_MODAL_NAVIGATOR || stateOrRoute.name === NAVIGATORS.LEFT_MODAL_NAVIGATOR)) {
```

Comment:
> `goBack` function contains logic for the RHP (isFirstRouteInNavigator). We should account for the LHP too

@s77rt I've modified this condition to check it

Reply:
But that does not seem enough. In goBack there is another check for the root name

Reply:
I've added the additional check